packages/core/minos-microservice-common/minos/common/configuration/config.py

```diff
@@ -29,6 +29,7 @@
 SAGA = namedtuple("Saga", "storage")
 REST = namedtuple("Rest", "host port")
 REPOSITORY = namedtuple("Repository", "database user password host port")
+QUERY_REPOSITORY = namedtuple("QueryRepository", "database user password host port")
```

Comment:
Could you remove the `QUERY_REPOSITORY` namedtuple? As it has the same structure as the `REPOSITORY` one you can share it for both config sections.